### PR TITLE
load main

### PR DESCRIPTION
**Pull request by AbrahamAlcaina**
Send events to server

### CODE REVIEW

File: client/src/js/states/loading.js [hunk 1]
```diff
@@ -32,18 +32,18 @@ class Loading extends Phaser.State {
         this.game.load.image("rules_rock_papaer_scissors_lizard_spock", require("../../assets/rules/rock_papaer_scissors_lizard_spock.jpg"));
 
         const loadCheck = () => {
-			if (this.game.load.hasLoaded) {
-				this.game.load.onLoadComplete.remove(loadCheck, this);
-				this.game.state.start('main');
```
Comment (bernatmv) [inline client/src/js/states/loading.js:37]: why do you remove this?

File: client/src/js/states/loading.js [hunk 2]
```diff
@@ -32,18 +32,18 @@ class Loading extends Phaser.State {
...
 
-        this.game.load.onFileComplete.add(() => this.loadUpdate());
-		this.game.load.onPackComplete.add(() => this.loadUpdate());
-		this.game.load.onLoadComplete.add(() => {
+        this.game.load.onFileComplete.add(loadCheck);
+        this.game.load.onPackComplete.add(loadCheck);
+        this.game.load.onLoadComplete.add(() => {
             preloadBar.text = this.game.load.progress + '%';
+            //TODO REMOVE
+            setTimeout(()=>this.game.state.start('main'), 1000);
```
Comment (bernatmv) [inline client/src/js/states/loading.js:46]: this is not needed if we start main after the loading process

File: client/src/js/actions.js [hunk 3]
```diff
@@ -1,3 +1,12 @@
+export const joinGame = (gameId, name, id) => ({
+    type: 'joinGame',
+    payload: {
+        gameId,
+        name,
+        fbid: id
+    }
+});
```
Comment (bernatmv) [inline client/src/js/actions.js:8]: what will be the flow for joining a game? I was thinking about providing a URL that will be posted to someone chat

**Review by bernatmv — CHANGES_REQUESTED** [review]
Some doubts regarding this PR

File: client/src/js/actions.js (hunk 3 above)
Comment (AbrahamAlcaina) [reply client/src/js/actions.js:8]: In any case we need to send the message of join to the server

File: client/src/js/states/loading.js (hunk 1 above)
Comment (AbrahamAlcaina) [reply client/src/js/states/loading.js:37]:
```
        this.game.load.onFileComplete.add(loadCheck);
        this.game.load.onPackComplete.add(loadCheck);
        this.game.load.onLoadComplete.add(() => {
            preloadBar.text = this.game.load.progress + '%';
            //TODO REMOVE
            setTimeout(()=>this.game.state.start('main'), 1000);
        }, this);
```
Seems that the bug of phaser is solved.
I create a handle for files other for packs and the final one for onload complete bug.

**Review by bernatmv — APPROVED** [review]

**Pull request merged**